diff --git a/crates/atuin-history/src/stats.rs b/crates/atuin-history/src/stats.rs
index b73a5dbb..9527ef1b 100644
--- a/crates/atuin-history/src/stats.rs
+++ b/crates/atuin-history/src/stats.rs
@@ -157,9 +157,9 @@ pub fn pretty_print(stats: Stats, ngram_size: usize) {
             .collect::<Vec<_>>()
             .join(" | ");
 
-        println!(
-            "{ResetColor}] {gray}{count:num_pad$}{ResetColor} {bold}{formatted_command}{ResetColor}"
-        );
+        println!(
+            "{ResetColor}] {count:num_pad$} {bold}{formatted_command}{ResetColor}"
+        );
     }
     println!("Total commands:   {}", stats.total_commands);
     println!("Unique commands:  {}", stats.unique_commands);
