diff --git a/internal/cmd/mod.go b/internal/cmd/mod.go
index 41b5dee..8429648 100644
--- a/internal/cmd/mod.go
+++ b/internal/cmd/mod.go
@@ -24,7 +24,7 @@ func Main(env Env, args []string, modBashPath string, modStdlib string, modVersi
 
 	err := CommandsDispatch(env, args)
 	if err != nil {
-		fmt.Fprintf(os.Stderr, errorColor+"%sdirenv: error %v%s", errorColor, err, clearColor)
+		fmt.Fprintf(os.Stderr, errorColor+"%sdirenv: error %v%s\n", errorColor, err, clearColor)
 	}
 	return err
 }
